CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
+* compute: added `external_managed_backend_bucket_migration_state` and `external_managed_backend_bucket_migration_testing_percentage` to `google_compute_global_forwarding_rule` resource. ([#9985](https://github.com/hashicorp/terraform-provider-google-beta/pull/9985))
+* compute: added `external_managed_migration_state` and `external_managed_migration_testing_percentage` to `google_compute_backend_service` resource. ([#9985](https://github.com/hashicorp/terraform-provider-google-beta/pull/9985))
+* compute: added `force_attach` field to `boot_disk` and  `attached_disk` of  `google_compute_instance`  resource ([#9999](https://github.com/hashicorp/terraform-provider-google-beta/pull/9999))
+* compute: added `numeric_id` to `google_compute_instance_template` resource ([#9975](https://github.com/hashicorp/terraform-provider-google-beta/pull/9975))
+* compute: added the numeric id as `generated_id` attribute to the `google_compute_network_endpoint_group` ([#9984](https://github.com/hashicorp/terraform-provider-google-beta/pull/9984))
+* compute: added update support for `load_balancing_scheme` in `google_compute_backend_service` and `google_compute_global_forwarding_rule` resources to allow migrating between classic and global external ALB ([#9985](https://github.com/hashicorp/terraform-provider-google-beta/pull/9985))
+* container: added `in_transit_encryption_config` field in `google_container_cluster` resource ([#9972](https://github.com/hashicorp/terraform-provider-google-beta/pull/9972))
+* container: allowed in-place update `node_config.windows_node_config` field in `google_container_cluster` and `google_container_node_pool` resource ([#9986](https://github.com/hashicorp/terraform-provider-google-beta/pull/9986))
+* container: allowed in-place update for `node_config.storage_pools` field in `google_container_cluster` and `google_container_node_pool` resourcee ([#9967](https://github.com/hashicorp/terraform-provider-google-beta/pull/9967))
+* dialogflowcx: added `event_handlers.trigger_fulfillment.enable_generative_fallback` field to `google_dialogflow_cx_flow` resource ([#9958](https://github.com/hashicorp/terraform-provider-google-beta/pull/9958))
+* dialogflowcx: added `gen_app_builder_settings` field to `google_dialogflow_cx_agent` resource ([#9971](https://github.com/hashicorp/terraform-provider-google-beta/pull/9971))
+* iambeta: added `mode`, `inline_certificate_issuance_config`, and `inline_trust_config` fields to `google_iam_workload_identity_pool` resource ([#9990](https://github.com/hashicorp/terraform-provider-google-beta/pull/9990))
+* vmwareengine: increased `google_cloud_vmwareengine_private_cloud` timeout to 6 hours. ([#9974](https://github.com/hashicorp/terraform-provider-google-beta/pull/9974))
+
+BUG FIXES:
+* compute: added global retry for "resourceNotReady for Networks" 400 errors ([#9970](https://github.com/hashicorp/terraform-provider-google-beta/pull/9970))
```

Comment:
What resource(s)?

Reply:
I think this is adding a global retry that applies to all resources